packages/nextjs-template/pages/_app.tsx

```diff
@@ -48,8 +51,22 @@ function AppContainer(appProps: AppProps & { pageProps: PageProps }) {
     const logLevel = getLogLevel();
     setLogLevel(logLevel);
   }, []);
+
+  const router = useRouter();
+
   const defaultTheme = ConfigUtils.getPublishing(config).theme || Theme.LIGHT;
   logger.info({ ctx: "enter", defaultTheme });
+  if (router.pathname === "/refs/[id]") {
+    logger.info({ appProps });
```

Comment:
appProps can be huge. lets not log unless you need specific information from it or have it as a debug log

Reply:
I just removed it